### DISCUSSION TITLE
Error in urequests.

### DISCUSSION THREAD

**Discussion by whatbeato** [micropython/micropython#9555] [Libraries & Drivers]
I'm getting this error while trying to start my webclock.

`Traceback (most recent call last):

  File "<stdin>", line 1, in <module>

  File "<string>", line 57, in <module>

  File "/lib/urequests.py", line 112, in get

  File "/lib/urequests.py", line 60, in request

OSError: (-256, 'CONN_LOST')`

My board is the esp8266, with an OLED screen.

If anyone knows the problem, I would appreciate the help :)

Thanks in advance,
glxxyx

**Comment by steveh127** [comment]
I think we need some more detail. ESP8266 is the mcu not the board and without at least the minimal set of code that's failing diagnosis is next to impossible.
